src/UI/Button.elm

```diff
@@ -340,269 +578,192 @@ type alias ButtonTheme =
     }
 
 
-styleAttrs : Button msg -> List (Attribute msg)
-styleAttrs ((Button _ opt) as btn) =
-    let
-        theme =
-            colorHelper btn
-
-        hover ( bg, border, text ) =
-            Element.mouseOver
-                [ Background.color bg
-                , Border.color border
-                , Font.color text
-                ]
-                :: Element.colorTransition 100
-
-        normal ( bg, border, text ) =
-            [ Background.color bg
-            , Border.color border
-            , Border.width <| borderWidth opt.size
-            , Font.color text
-            ]
-
-        colors { primary, text, outlinedBg } =
-            if isOutlined btn then
-                ( Palette.toElColor outlinedBg
-                , Palette.toElColor primary
-                , Text.ColorPalette primary
-                    |> Text.fontColor
-                    |> Maybe.withDefault
-                        Palette.gray.middle
-                )
-
-            else
-                ( Palette.toElColor primary
-                , Palette.toElColor primary
-                , text
-                    |> Text.fontColor
-                    |> Maybe.withDefault
-                        Palette.gray.middle
-                )
-    in
+themeApply : ButtonTheme -> List (Attribute msg)
```

Comment:
Looking at the type signature, I would say that this function should be called `themeToAttributes ` and not the other one.

Reply:
That's what the function below is called :eyes: 

I'll do something anyway...